File: docs/source/examples/Relationships/ToMany/ToMany.swift
class Person: Object {
    dynamic var name: String = ""
    dynamic var birthdate: Date = Date(timeIntervalSince1970: 1)
    let dogs = List<Dog>()
}

class Dog: Object {
    dynamic var name: String = ""
    dynamic var age: Int = 0
    dynamic var breed: String? = nil
}

File: docs/source/examples/Relationships/ToMany/ToMany.kt
class Person: RealmObject {
    var name: String = ""
    var birthdate: Date = Date(milliseconds: 1)
    var dogs = RealmList<Dog>()
}

class Dog: RealmObject {
    var name: String = ""
    var age: int = 0
    var breed: String? = null
}
